app/models/action.js

```diff
@@ -10,19 +10,20 @@ export default class ActionModel extends Model {
   // One Project has Many Actions
   @belongsTo('project') project;
 
-  // Many Actions to Many Recommendations
-  // Participants can submit ONE recommendation per action or ONE recommendation for all actions in a project
-  @hasMany('recommendation', { inverse: 'actions' }) recommendations;
+  // ONE disposition has many action
+  @hasMany('action') actions;
```

Comment:
oops, did we mean to write `@hasMany('disposition') dispositions`?

Reply:
yes! will fix